packages/repository-tests/src/types.repository-tests.ts

```diff
@@ -64,6 +64,14 @@ export interface CrudFeatures {
    * Default: `true`
    */
   supportsInclusionResolvers: boolean;
+
+  /**
+   * Does the connector require the revision token for replacement operations?
+   * (Cloudant)
+   *
+   * Default: `false`
+   */
+  revisionTokenRequired: boolean;
```

Comment:
Nitpick - feel free to ignore.

The `_rev` value is not always required, is it? For example, when creating a new model instance, the `_rev` must not be provided, at least AFAIK.

Maybe `hasRevisionToken` would be a better name?

Reply:
make sense